example/Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: dredd
+
+hooks/hooks:
+	cd hooks/; go build; cd ..;
+
+server:
+	go build -o server
+
+dredd: hooks/hooks server
+	~/Code/dredd/bin/dredd api.apib http://localhost:8080 --language go --server ./server --hookfiles hooks/hooks
```

Comment:
@snikch see my TODO.  The PR was just merged but I need to wait for a release since npm is lacking for installing packages via github urls.